llama_index/embeddings/clarifai.py

```diff
@@ -115,7 +115,6 @@ def _embed(self, sentences: List[str]) -> List[List[float]]:
 
     def _get_query_embedding(self, query: str) -> List[float]:
         """Get query embedding."""
-        query = self._format_query_text(query)
```

Comment:
There is no `_format_query_text` in `ClarifaiEmbedding` at this point in time, so this was a latent bug